demosplan/DemosPlanCoreBundle/Resources/client/scss/objects/_form.scss

```diff
@@ -293,7 +293,7 @@ input.disabled {
     // The height is calculated by adding $dp-button-font-size (which is used for
     // button line-height), vertical padding and 2px border.
     $element-height: $dp-button-font-size + $dp-form-element-spacing-vertical * 2 + 2px;
-
+// 15px + 2 * 13px + 2px = 43px
```

Comment:
indentation. And maybe its better positioned above the line?

Reply:
Thanks for finding this, I did not mean to commit the comment :sweat_smile: Removed in 0ec4a4f0075b745aaa1e80a3dadff45cfd11c502